src/main/java/io/github/queerbric/queerbrics/datatypes/item/QueerBrickItem.java

```diff
@@ -26,6 +26,7 @@ public QueerBrickItem() {
 			throwableBrickEntity.setItem(itemStack);
 			throwableBrickEntity.setProperties(user, user.pitch, user.yaw, 0.0F, 1.5F, 1.0F);
 			world.spawnEntity(throwableBrickEntity);
+			itemStack.decrement(1);
```

Comment:
This should not be decremented if the player has creative-mode abilites

Reply:
This is decrement's vanilla behavior, it will not decrement in creative